XenModel/Messages.resx

```diff
@@ -10753,7 +10750,16 @@ This will cancel the rolling pool upgrade process and may leave your system in a
   <data name="ROLLING_UPGRADE_ERROR_POOL_ONE" xml:space="preserve">
     <value>The following error occurred while rolling pool upgrade was in progress:</value>
   </data>
+  <data name="ROLLING_UPGRADE_EXTRAS_PAGE_TEXT" xml:space="preserve">
+    <value>Extras</value>

```

Comment:
I'm having second thoughts about `Extras`, it's not very descriptive of what it is; perhaps use `Upgrade Options`? Other suggestions?

Reply:
`Upgrade Options` sounds better than `Extras`